Musigwa/RNLevelUp
# Configure ESLint and integrate HoundCI

#### What does this PR do?

- It sets up the linter(ESLint) and add HoundCI bot as a style checker
#### Description of Task to be completed?

- This is to configure the ESLint and HoundCI as the code style checker with react/react-native best practice rules. I'm using `Airbnb style guide`

#### How should this be manually tested?
- After cloning the repository, check out `ch-setup-linter` branch
- Checkout the `README.md` file for set up instructions
- Run `npm install` to install the project's dependencies
- Run `npm test` to runs all test scenarios and see the results in from the terminal
- Try modifying a file using any style that goes against the rules, and you shall see the `eslint` warning/errors
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- None
